src/ButtonFolder/ButtonFolder.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import "../ButtonFolder/ButtonFolder.css";
```

Comment:
Путь может быть короче:
```suggestion
import "./ButtonFolder.css";
```

Исправь, пожалуйста, так же в других местах.